tools/flaky_test_monitor/level1/process_summary1_results_test.go

```diff
@@ -6,89 +6,142 @@ import (
-		"10 count some failures":    "test-result-crypto-hash-10-count-fail.json",
+		"2 count all pass": {
+			ExpectedLevel1Summary: testdata.GetTestData_Level1_2CountPass(),
+			RawJSONTestRunFile:    "test-result-crypto-hash-2-count-pass.json",
+		},
+
+		// raw results generated with: go test -json -count 1 --tags relic ./utils/unittest/...
+		"10 count all pass": {
+			ExpectedLevel1Summary: testdata.GetTestData_Level1_10CountPass(),
+			RawJSONTestRunFile:    "test-result-crypto-hash-10-count-pass.json",
+		},
+
+		// raw results generated with: go test -json -count 1 --tags relic ./utils/unittest/...
+		"10 count some failures": {
+			ExpectedLevel1Summary: testdata.GetTestData_Level1_10CountSomeFailures(),
+			RawJSONTestRunFile:    "test-result-crypto-hash-10-count-fail.json",
+		},
 
 		// nil tests - tests below don't generate pass/fail result due to `go test` bug
 		// with using `fmt.printf("log message")` without newline `\n`
 
 		// raw results generated with: go test -v -tags relic -count=1 -json ./model/encodable/. -test.run TestEncodableRandomBeaconPrivKeyMsgPack
 		// this is a single unit test that produces a nil test result
-		"1 count single nil test": "test-result-nil-test-single-1-count-pass.json",
+		"1 count single nil test": {
+			ExpectedLevel1Summary: testdata.GetTestData_Level1_1CountSingleNilTest(),
+			RawJSONTestRunFile:    "test-result-nil-test-single-1-count-pass.json",
+		},
 
 		//raw results generated with: go test -v -tags relic -count=5 -json ./model/encodable/. -test.run TestEncodableRandomBeaconPrivKeyMsgPack
 		//multiple nil tests in a row
-		"5 nil tests in a row": "test-result-nil-test-single-5-count-pass.json",
+		"5 nil tests in a row": {
+			ExpectedLevel1Summary: testdata.GetTestData_Level1_5CountSingleNilTest(),
+			RawJSONTestRunFile:    "test-result-nil-test-single-5-count-pass.json",
+		},
 
 		//normal test at the end of a test run with multiple nil tests in front of it
-		"4 nil tests in a row, 1 normal test": "test-result-nil-test-single-5-count-4-nil-1-normal-pass.json",
+		"4 nil tests in a row, 1 normal test": {
+			ExpectedLevel1Summary: testdata.GetTestData_Level1_5CountMultipleNilTests(),
+			RawJSONTestRunFile:    "test-result-nil-test-single-5-count-4-nil-1-normal-pass.json",
+		},
 
 		// raw results generated with: go test -v -tags relic -count=3 -json ./model/encodable/.
 		// group of unit tests with a single nil test result
-		"3 count nil test with normal tests": "test-result-nil-test-others-normal-3-count-pass.json",
+		"3 count nil test with normal tests": {
+			ExpectedLevel1Summary: testdata.GetTestData_Leve1_3CountNilWithNormalTests(),
+			RawJSONTestRunFile:    "test-result-nil-test-others-normal-3-count-pass.json",
+		},
+	}
+
+	require.NoError(t, os.Setenv("COMMIT_DATE", "2021-09-21T18:06:25-07:00"))
+	require.NoError(t, os.Setenv("COMMIT_SHA", "46baf6c6be29af9c040bc14195e195848598bbae"))
+	require.NoError(t, os.Setenv("JOB_STARTED", "2021-09-21T21:06:25-07:00"))
```

Comment:
nit: define these values as variables?

Reply:
extracted to `testdata` package

https://github.com/onflow/flow-go/pull/2102/commits/e0672bd5a3e9c83c1446787cfefcbd583630e55b